### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by JiejueAwA** [amll-dev/applemusic-like-lyrics#246] [Q&A] [answered]
在每首歌的歌词辉光部分，字体会抖动放大，貌似与字号或其他问题所造成的
还有不知道为什么在关闭处，飞几次鼠标，然后右上角出现一个长条，无法关闭
还有关闭键需要点击两次才能关闭

**Comment by MianSoft** [comment, marked as answer]
辉光逐字发光升起是按照 AM 去做的
抖动是网易云问题,建议使用外置播放器
